deploy/crds/toolchain_v1alpha1_useraccount.yaml

```diff
@@ -58,34 +58,12 @@ spec:
             nsLimit:
               description: The namespace limit name
               type: string
-            nsTemplateSet:
-              description: Namespace template set
-              properties:
-                namespaces:
-                  description: The namespace templates
-                  items:
-                    properties:
-                      revision:
-                        description: The revision of the corresponding template
-                        type: string
-                      template:
-                        description: Optional field. Used to specify a custom template
-                        type: string
-                      type:
-                        description: 'The type of the namespace. For example: ide|cicd|stage|default'
-                        type: string
-                    required:
-                    - type
-                    - revision
-                    type: object
-                  type: array
-                tierName:
-                  description: The name of the tier represented by this template set
-                  type: string
-              required:
-              - tierName
-              - namespaces
+            nsTemplateSetName:
+              description: Reference to NSTemplateSet for this user
               type: object
+            tierName:
```

Comment:
`tierName` is part of `NSTemplateSet`. Why do we also need it in `UserAccount`?

Reply:
I think I know why you added it but see my comments above.

Reply:
@alexeykazakov NSTemplateSet is created by UserAccountCtrl, to set NSTemplateSet.TierName we need UserAccount.TierName.